app/controllers/concerns/sentry_controller_logging.rb

```diff
@@ -24,8 +24,14 @@ def user_context
 
   def tags_context
     { controller_name: }.tap do |tags|
-      if current_user.present?
-        sign_in = current_user.respond_to?(:identity) ? current_user.identity.sign_in : current_user.sign_in
```

Comment:
The previous check for identity method was correct and needs to be kept in the nil check chain/block likely because `AccreditedRepresentativePortal::RepresentativeUser` expected it that way